libs/features/icons/icons.js

```diff
@@ -34,7 +34,17 @@ export const fetchIcons = (config) => new Promise(async (resolve) => {
   resolve(fetchedIcons);
 });
 
-function decorateToolTip(icon) {
+function checkIfIconIsTooltip(icon) {
+  const wrapper = icon.closest('em');
+  if (!wrapper) return null;
+
+  const conf = wrapper.textContent.includes('|');
+  if (!conf) return null;
+
+  return [...icon.classList].some((cls) => cls.includes('tooltip'));
+}
+
+function decorateToolTip(icon, iconName) {
```

Comment:
If this is a one-liner that isn't re-used, I'd not make it a helper function and move it closer to where its actual used.

```suggestion
function decorateToolTip(icon, iconName) {
    const hasTooltip = icon.closest('em')?.textContent.includes('|') && [...icon.classList].some((cls) => cls.includes('tooltip'))
    if(!hasTooltip) return
```